src/actions/link.actions.test.js

```diff
@@ -1,175 +1,176 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
You can remove it now that the rule was added by: https://github.com/Talend/tools/pull/15